webpack.config.js

```diff
@@ -54,7 +54,7 @@ module.exports = {
                 // .babelrc is used for transpiling src/ into lib/ in the prepublish
                 // phase, see package.json
                 options: {
-                    presets: ['es2015'],
+                    presets: [['es2015', { modules: false } ]],
```

Comment:
Babel support for `export * from 'foo'` seems broken, so I let webpack deals with that.